pallets/parachain-staking/src/lib.rs

```diff
@@ -201,6 +201,7 @@ pub mod pallet {
 	}
 
 	#[derive(Encode, Decode, RuntimeDebug, TypeInfo)]
+	/// DEPRECATED
```

Comment:
Would a deprecated attribute be useful here? https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-deprecated-attribute

Reply:
Also, a note on what has superseded it would be helpful.

Reply:
> Would a deprecated attribute be useful here? https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-deprecated-attribute

No, we don't need warnings emitted by using the struct especially because we will use it in the migration. From that link: `"The deprecated attribute marks an item as deprecated. rustc will issue warnings on usage of #[deprecated] items. "

> Also, a note on what has superseded it would be helpful.

will do